apps/docs/src/routes/examples/camera-followobject.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Follow Object
+---
+
+<script lang="ts">
+import Wrapper from '$examples/camera/followobject/wrapper.svelte'
```

Comment:
I think this is the problem. Unfortunately I'm not on my dev machine and only on an iPad using Codespaces but it seems to be the culprit. TypeScript in .md files would be awesome.

Reply:
@DefinitelyMaybe

Reply:
Seems to work now :)

Reply:
My bad, and thank you. totally agree that typescript or a lsp picking that up would save both of us the trouble.